addons/skin.estuary/xml/SkinSettings.xml

```diff
@@ -67,6 +67,13 @@
 					<onclick>Skin.SelectBool($LOCALIZE[31024], 38018|circle_userrating, 563|circle_rating, 16018|)</onclick>
 					<label2>$VAR[RatingSettingLabel2Var]</label2>
 				</control>
+                                <control type="button" id="707">
+                                        <label>$LOCALIZE[31164]</label>
+                                        <include>DefaultSettingButton</include>
+                                        <onclick>Skin.SelectBool(31164, 31165|show_profilename, 31166|show_profileavatar, 16018|)</onclick>
```

Comment:
please add show_none at the end. SelectBool() needs a setting id for each option.
i'm kinda surprised it even works as is :-)

Reply:
done